corehq/apps/reports/filters/users.py

```diff
@@ -190,6 +209,15 @@ def user_tuple(self, u):
         return (uid, name)
 
 
+class UsersUtilsAllUsers(EmwfUtilsAllUsers):
+
+    def user_tuple(self, u):
+        user = util._report_user_dict(u)
+        uid = "%s" % user['user_id']
+        name = "%s" % user['username_in_report']
+        return (uid, name)
```

Comment:
What's this one for?  I didn't think there'd be anything which displays only users and wants to include deactivated ones.

Reply:
Good point, I think I was just on a subclassing spree:
https://github.com/dimagi/commcare-hq/pull/21564/commits/0ba1cc077c42b11d14e69f85a3160345d254559d